velox/docs/functions/presto/regexp.rst

```diff
@@ -32,6 +32,12 @@ limited to 20 different expressions per instance and thread of execution.
         SELECT like('abc', '%b%'); -- true
         SELECT like('a_c', '%#_%', '#'); -- true
 
+    String sequence search: There are some patterns that are equivalent to simple
```

Comment:
It would be better to embed the new information into the existing flow.

> Not all patterns require compilation of regular expressions. Patterns ....

The new pattern needs to added to this list. The separate paragraph then can be about the limitations that apply to these special patterns. I.e. We would clarify separately whether special patterns allow for custom character or not.

Also, please, generate the docs to make sure they look nice.

Reply:
![image](https://github.com/user-attachments/assets/bc7a7f41-28fb-4bf5-b5f3-a8df2e226d17)

I am not sure if I understand your meaning, add a list for all fast path in velox.

@mbasmanova  Could you look at this again ?